app/src/main/java/githubuser/model/GithubUsers.java

```diff
@@ -0,0 +1,25 @@
+package githubuser.model;
+
+import com.google.gson.annotations.SerializedName;
+
+public class GithubUsers {
+
+    @SerializedName("userName")
```

Comment:
the SerializedName should match the name of the data that will be retrieved from the google API.